bugzilla.pl

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/perl
```

Comment:
does this mean bugzilla will no longer run with taint checks enabled?  it's saved our bacon on multiple occasions before.